index.js

```diff
@@ -92,7 +92,7 @@ class ResembleHelper extends Helper {
               fs.mkdirSync(getDirName(this.diffFolder + diffImage));
             }
             fs.writeFileSync(`${this.diffFolder + diffImage}.png`, data.getBuffer());
-            const diffImagePath = path.join(process.cwd(), `${this.diffFolder + diffImage}.png`);
+            const diffImagePath = path.join(`${this.diffFolder + diffImage}.png`);
```

Comment:
We don't need the `join` at all here, right?

Reply:
+1